svf-llvm/lib/LLVMModule.cpp

```diff
@@ -827,6 +830,94 @@ void LLVMModuleSet::buildFunToFunMap()
             decls.push_back(decl);
         }
     }
+
+    /*
```

Comment:
put these comments before the function.

Fix AppDef -> ExtDef/ExtDef -> AppDef (overwrite).

Give each case an example.